Procurement/Utility/StashHelper.cs

```diff
@@ -99,22 +99,40 @@ private static System.Drawing.Bitmap buildImage(Tab tab, List<System.Drawing.Bit
 
                     BitmapImage bitmapclone = (BitmapImage)bitmap.Clone();
                     bitmap = null;
-                    imageCache.Add(key, new CroppedBitmap(bitmapclone, new Int32Rect(0, offset, (int)bitmapclone.Width, 26)));
+
+                    Int32Rect croppingRectangle = new Int32Rect();
+                    if (offset + 26 > bitmapclone.Height)
```

Comment:
26 is a magic int which is being used later on, constant please :)

Reply:
Done.